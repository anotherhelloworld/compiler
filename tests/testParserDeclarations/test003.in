const
a: integer = 10;
c: double = 0;
b = 250;
