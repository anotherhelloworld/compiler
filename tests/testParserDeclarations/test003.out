Const   a
   Type   integer
   10

Const   c
   Type   double
   0

Const   b
   250

